users/migrations/0002_static_data.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from django.db import migrations, transaction
+
+
+def create_initial_users(apps, schema_editor):
+    Profile = apps.get_model('users', 'Profile')
+    User = apps.get_model('auth', 'User')
+    user_ofek = User(username='ofek', password='qaz[;.123')
+    user_danny = User(username='danny', password='qaz[;.123')
+    user_rimon = User(username='rimon', password='qaz[;.123')
+    user_rotem = User(username='rotem', password='qaz[;.123')
+    user_saar = User(username='Saar', password='qaz[;.123')
+    user_tal = User(username='Tal', password='qaz[;.123')
+    user_ofek.save()
+    user_danny.save()
+    user_rimon.save()
+    user_rotem.save()
+    user_saar.save()
+    user_tal.save()
+
+    profile_data = [
+        (user_ofek, datetime(2020, 5, 17), '050-123456789'),
+        (user_danny, datetime(2020, 5, 17), '050-223456789'),
+        (user_rimon, datetime(2020, 5, 17), '050-323456789'),
+        (user_rotem, datetime(2020, 5, 17), '050-423456789'),
+        (user_saar, datetime(2020, 5, 17), '050-523456789'),
+        (user_tal, datetime(2020, 5, 17), '050-623456789'),
+    ]
+    with transaction.atomic():
+        for user, dob, phone_number in profile_data:
+            profile = Profile(user=user, date_of_birth=dob, phone_number=phone_number)
+            profile.save()
```

Comment:
maybe you can do it simply with one loop like:
```suggestion
    profile_data = [
        ('Ofek', 'qaz[;.123', datetime(2020, 5, 17), '050-123456789'),
        ('Danny', 'qaz[;.123', datetime(2020, 5, 17), '050-223456789'),
        ('Rimon', 'qaz[;.123', datetime(2020, 5, 17), '050-323456789'),
        ('Rotem', 'qaz[;.123', datetime(2020, 5, 17), '050-423456789'),
        ('Saar', 'qaz[;.123', datetime(2020, 5, 17), '050-523456789'),
        ('Tal', 'qaz[;.123', datetime(2020, 5, 17), '050-623456789'),
    ]

    with transaction.atomic():
        for username, password, dob, phone_number in profile_data:
            user = User(username=username, password=password)
            user.save()
            profile = Profile(user=user, date_of_birth=dob, phone_number=phone_number)
            profile.save()

```